doc/whats_new/v0.21.rst

```diff
@@ -613,6 +613,12 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.svm`
 ..................
 
+- |Enhancement| :class:`svm.SVC` and :class:`NuSVC` now accept a `break_ties`
+  parameter. This parameter results in `predict` breaking the ties according
+  to the confidence values of `decision_function`, if
+  `decision_function_shape`='ovr', and the number of target classes > 2.
```

Comment:
I think `decision_function_shape` should be in double quotes. If you look at the rendered whats new, it's causing a formatting issue.